packages/form-store/src/index.ts

```diff
@@ -11,9 +11,12 @@ export const validating: ValidationResultValidating = { status: "validating" }
 
 export class FormStore<T> {
 	canSubmit$: Observable<boolean>
+	private readonly nonce$ = Atom.create(0)
```

Comment:
What about idleCache? So you won't be need to `nonce$.modify`, jus `nonce$.set(idleCache)`